packages/renderer/src/lib/container/CreateContainerFromExistingImage.svelte

```
@@ -315,9 +316,13 @@ async function searchFunction(value: string): Promise<void> {
   // do not search for images if no connection is selected
   if (!selectedProviderConnection) return;
 
+  const requestId = ++searchRequestId;
   value = value.trim();
```

Biome (2.5.3)

[error] Assigning a function parameter is confusing.

(lint/style/noParameterAssign)